easybuild/easyconfigs/m/Mauve/Mauve-2.3.1.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = "Tarball"
+
+name = 'Mauve'
+version = '2.3.1'
+
+homepage = 'http://gel.ahabs.wisc.edu/mauve'
+description = """ Mauve is a system for efficiently constructing multiple genome 
+ alignments in the presence of large-scale evolutionary events such as rearrangement 
+ and inversion. Multiple genome alignment provides a basis for research into comparative 
+ genomics and the study of evolutionary dynamics. """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://gel.ahabs.wisc.edu/mauve/downloads/']
+sources = ['%(namelower)s_linux_%(version)s.tar.gz']
+
+dependencies = [ ('Java', '1.7.0_21') ]
```

Comment:
possible to bump minor release tag? 

remove extra line at the end, overall ok here